build.psm1

```diff
@@ -1237,13 +1208,12 @@ function Start-PSBootstrap {
             $DotnetArguments = @{ Channel=$Channel; Version=$Version; NoSudo=$NoSudo }
             Install-Dotnet @DotnetArguments
         }
-        else
-        {
+        else {
             log "dotnet is already installed.  Skipping installation."
         }
 
-        # Install for Windows
-        if ($Environment.IsWindows) {
+        # Install Windows dependencies for building native binaries or creating Appx package
+        if ($Environment.IsWindows -and $Package) {
             $machinePath = [Environment]::GetEnvironmentVariable('Path', 'MACHINE')
             $newMachineEnvironmentPath = $machinePath
 
```

Comment:
@joeyaiello are you ok if we remove Appx support?  Seems like this is something we can bring back later.